Week-2/scripts/RenderContent.js

```diff
@@ -0,0 +1,76 @@
+function setRightContent(fullContainer,container){
+  let right_container = document.createElement('div'),
+  titleSelectCategory = document.createElement('h4'),
+  select = document.createElement('select'),
+  subscribeTitle = document.createElement('h4'),
+  check=document.createElement('div'),
+  input = document.createElement('input'),
+  btn = document.createElement('button');
+
+  right_container.setAttribute('class', 'right-container');
+  titleSelectCategory.textContent = selectCategoryText;
+  select.setAttribute('id','ddlSelect');
+  select.options.add( new Option(defaultSelectChannelText,null));
+  let data=[];
+  for(item in article_data){
+    if(data.indexOf(article_data[item].postCategory)!==-1){
+      continue;
+    }
+    else{
+      select.options.add( new Option(article_data[item].postCategory,article_data[item].postCategory) );
+      data.push(article_data[item].postCategory);
+    }
+}
+  subscribeTitle.textContent = subscribeText;
+  check.setAttribute('class','check');
+  input.setAttribute('id','email');
+  input.type = 'text';
+  btn.innerHTML=subscribeText;
+  btn.setAttribute('id','btn-email');
+  btn.addEventListener('click', emailSave);
+  check.appendChild(input);
+  check.appendChild(btn);
+  right_container.appendChild(titleSelectCategory);
+  right_container.appendChild(select);
+  right_container.appendChild(subscribeTitle);
+  right_container.appendChild(check);
```

Comment:
few places same dom manipulation function used, this can be clubbed by making a few common functions. this can reduce a lot of code lines.
discuss with me if you are not clear what i am telling